app/tut/ga/an-staitse.md

```diff
@@ -0,0 +1,256 @@
+---
+title: An Stáitse
+prev: cad-e-riomhchlaru
+prev-text: Cad é Ríomhchlárú?
+---
+
+# Gníomhartha
+
+Le tosú ag tarraingt cruthanna ar an stáitse, is gá dúinn féach ar **gníomhartha**.
+
+Chonaiceamar cúpla gníomhartha cheana féin, ba gníomh é `scríobh`{.setanta}, chomh maith le
+`codladh`{.setanta}.
+
+Is luachanna speisialta iad gníomhartha a sheasann do rudaí casta is féidir leis an ríomhaire a
+dhéanamh. Sa chás `gníomh`{.setanta} mar shampla, is é an rud casta ná téacs a scríobh ar an gconsól
+agus sa chás `codladh`{.setanta}, is é ag fanacht ar feadh am éigin.
+
+Amach anseo, feicfimid conas ár ngníomhartha féin a chruthú, ach anois feicfimid ar an caoi is
+féidir linn iad a úsáid.
+
+## Glaoigh Orm!
+
+Nuair a bainimid úsáid as gníomh, deirimid go bhfuilimid **ag glaoch** air. An cuimhin leat an caoi
+a d'úsáideamar an gníomh `scríobh`{.setanta}?
+
+```{.setanta .numberLines}
+scríobh("Scríobh mé ar an gconsól")
+```
+
+Nuair a léann [[léirmhínitheoir|interpreter]] *Setanta*
+"`scríobh("Scríobh mé ar an gconsól")`{.setanta}", glaonn sé ar an gníomh `scríobh`{.setanta} agus
+tugann sé an téacs `"Scríobh mé ar an gconsól"`{.setanta} dó.
+
+Tugaimid [["**argóint**"|argument]] ar an téacs idir na [[lúibíní|brackets]]. Sa chód thuas, is é
+`"Scríobh mé ar an gconsól"`{.setanta} an argóint do `scríobh`{.setanta}
+
+Sa chód a leanas, is é `2000`{.setanta} an argóint do `codladh`{.setanta}.
+
+```{.setanta .numberLines}
+codladh(2000)
+```
+
+Nuair a ritear an cód seo, tugann an ríomhaire `2000`{.setanta} don gníomh `codladh`{.setanta} agus
+ansin glaonn sé air. Ansin fanann an léirmhínitheoir ar feadh dhá soicind.
+
+### Níos mó argóintí
+
+Is féidir le roinnt gníomhartha i *Setanta* [[glacadh|accept]] le níos mó ná argóint amháin. Nuair a
+tugaimid níos mó ná argóint amháin do gníomh éigin, bainimid úsáid as [[camóg|comma]] ("`,`") chun
+iad a [[scaradh|separate]].
+
+Mar shampla, is féidir le `scríobh`{.setanta} an oiread argóintí agus is mian linn a ghlacadh, agus
+scríobhfaidh sé gach ceann ar an gconsól le spás eatarthu. Bain triail as anseo:
+
+{{{
+scríobh("An-chéad-argóint", "An-dara-argóint")
+}}}
+
+Ní féidir le gach gníomh an oiread argóintí agus is mian linn a ghlacadh. Glacann an gníomh
+`codladh` le argóint amháin, ní ghlacann sé le aon méid argóinte eile.
```

Comment:
le hargóint amháin

https://www.teanglann.ie/ga/gram/_p_n?p=le_prep&n=arg%c3%b3int_fem2